nservicebus/azure-servicebus/multiple-namespaces-support.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Multiple namespace support
+summary: Configuring Azure ServiceBus transport to support different partitioning strategies. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+reviewed: 2016-04-18
+---
+
+// INTRODUCTION HERE!!!
+
+To configure _namespace partitioning strategy_ properly users have to select desired `INamespacePartitioningStrategy` implementation and map related namespaces, as showed in the next snippets, for each supported strategies.
```

Comment:
> users have to select 

I can't recall doco talking about users. Perhaps this would be easier to change if we first have explanation (intro) about what Namespace Partitioning is.